src/main/java/com/arpnetworking/http/Routes.java

```diff
@@ -217,6 +245,7 @@ private String createTimerName(final HttpRequest request) {
     private static final ProcessorsV2Factory TELEMETRY_V2_FACTORY = new ProcessorsV2Factory();
     private static final String TELEMETRY_STREAM_V1_PATH = "/telemetry/v1/stream";
     private static final String TELEMETRY_STREAM_V2_PATH = "/telemetry/v2/stream";
+    private static final String SOURCE_PREFIX = "/source/";
```

Comment:
Ahh, if I read this correctly the endpoints under source are intended to capture more than just CollectD. I'm not wild about this generalization for exactly the reason you describe, that you cannot version collectd endpoint along with a standard set of other endpoints; namely our formats and say statsd. They will all be different. However, saying that versioning does not apply to them is also incorrect. 

So I will generalize my advice to say that the source prefix can remain as is, but the specific sources should be versioned. For example, "/v1/collectd". Likewise, this permits actor versioning, which is something we have not been particularly good at. 

At the moment all our versioning is done through the parser which couples all versions of a given parser together. By moving the versioning down a level we can separate the different parser versions ;) -- not possible with files, but very sweet with REST!